Eastern-Research-Group/expert-query
# Feature/204 swagger UI page

## Related Issues:
* [EQ-204](https://jira.epa.gov/browse/EQ-204)

## Main Changes:
* Added Swagger API documentation for the profile `/data` routes and there corresponding `/count` routes.
  * I left the contact email blank for now. Also most parameters still need descriptions, since we're still waiting on glossary/tooltip descriptions from EPA.
  * The configuration file is stored with the S3 content, so the filepath shown in the UI would be strange if displayed:
  
![swagger](https://user-images.githubusercontent.com/104089016/233712150-6db00e96-4c7a-4105-bf5e-1fb9862aba6c.png)

If we want this JSON file to be publicly accessible, we may need to move it to the `public` folder; currently, the link to the file is hidden.

## Steps To Test:
1. Go to the API docs page: http://localhost:3000/api-docs.
2. Confirm that all routes are properly documented, and that trying queries works as expected.
